torch/_export/converter.py

```diff
@@ -79,6 +155,8 @@ def __init__(
         ts_graph: Union[torch._C.Graph, torch._C.Block],
         param_names: Set[str],
         buffer_names: Set[str],
```

Comment:
can we get rid of `param_names` and `buffer_names` if there exists the `mod_to_param_and_buffer_map`?